reasoner/controller/ConjunctionController.java

```diff
@@ -107,12 +103,10 @@ protected void setUpUpstreamControllers() {
     abstract FunctionalIterator<Concludable> concludablesTriggeringRules();
 
     List<Resolvable<?>> plan(Set<Variable.Retrievable> boundVariables) {
-        if (!plans.containsKey(boundVariables)) {
-            List<Resolvable<?>> plan = Planner.plan(resolvables, new HashMap<>(), boundVariables);
-            plan.addAll(conjunction.negations());
-            plans.put(boundVariables, plan);
-        }
-        return plans.get(boundVariables);
+        ReasonerPlanner.Plan<Resolvable<?>> planObj = registry().planner().getPlan(conjunction, boundVariables);
+        List<Resolvable<?>> newPlan = planObj.planOrder();
```

Comment:
we do we need to expose a `Plan` object at all? can we just get the `planOrder` directly?